packages/wallet/package.json

```diff
@@ -14,8 +14,8 @@
     "build:typescript": "npx tsc -b",
     "build:webpack": "CI=false node scripts/build.js",
     "build": "run-s build:typescript build:webpack",
-    "lint:check": "eslint \"*/**/*.{js,ts,tsx}\" --cache",
-    "lint:write": "eslint \"*/**/*.{js,ts,tsx}\" --fix",
+    "lint:check": "eslint \"src/**/*.{js,ts,tsx}\" --cache",
+    "lint:write": "eslint \"src/**/*.{js,ts,tsx}\" --fix",
```

Comment:
Why do we not want to lint files that are not in `src`? We don't care how our config files look and act?

Reply:
We can change it to something like `{src/config}` but I feel it is better to be explicit vs implicit with these kinds of tools lest they blow up due to some file / folder that's full of files. It also mimics how it worked before.